### PR TITLE
mtx3: add getCurrentStatusMeter, getCurrentValues commands

### PR DESCRIPTION
**Pull request by aleks-and**

### CODE REVIEW

File: src/mtx3/commands/uplink/getCurrentStatusMeter.ts [hunk 1]
```diff
@@ -0,0 +1,305 @@
...
+
+import * as command from '../../../mtx/utils/command.js';
+import {READ_ONLY} from '../../../mtx/constants/accessLevels.js';
+import {IEventStatus, eventStatusMask} from '../../../mtx/utils/CommandBinaryBuffer.js';
+import CommandBinaryBuffer, {ICommandBinaryBuffer} from '../../utils/CommandBinaryBuffer.js';
+import * as bitSet from '../../../utils/bitSet.js';
+import * as types from '../../types.js';
+
+
+interface ICurrentTariffs {
```
Comment (DeadByElpy) [inline src/mtx3/commands/uplink/getCurrentStatusMeter.ts:70]:
```suggestion
interface ITariff {
```
?

File: src/mtx3/commands/uplink/getCurrentStatusMeter.ts [hunk 2]
```diff
@@ -0,0 +1,305 @@
...
+    /**
+     * Current tariff for the group of consumed energies (`1st` tariff grid).
```
Comment (DeadByElpy) [inline src/mtx3/commands/uplink/getCurrentStatusMeter.ts:72]: tautology

**Review by DeadByElpy — CHANGES_REQUESTED** [review]

File: src/mtx/utils/CommandBinaryBuffer.ts [hunk 3]
```diff
@@ -716,21 +716,84 @@ export interface IPackedEnergiesWithType {
...
+     */
+    PARAMETERS_UPDATE_LOCAL: boolean;
+
+    /**
+     * Meter program restart.
+     */
+    RESTART: boolean;
+
+    /**
+     * Incorrect password and block.
```
[the remark this answers is not in the document]
Comment (aleks-and) [reply src/mtx/utils/CommandBinaryBuffer.ts]: fixed

File: src/mtx/utils/CommandBinaryBuffer.ts [hunk 4]
```diff
@@ -716,21 +716,84 @@ export interface IPackedEnergiesWithType {
...
+     */
+    ERROR_ACCESS: boolean;
+
+    /**
+     * Time set.
```
[the remark this answers is not in the document]
Comment (aleks-and) [reply src/mtx/utils/CommandBinaryBuffer.ts]: fixed

File: src/mtx3/commands/uplink/getCurrentStatusMeter.ts (hunk 1 above)
Comment (aleks-and) [reply src/mtx3/commands/uplink/getCurrentStatusMeter.ts]: applied

File: src/mtx3/commands/uplink/getCurrentStatusMeter.ts (hunk 2 above)
Comment (aleks-and) [reply src/mtx3/commands/uplink/getCurrentStatusMeter.ts]: fixed

**Review by DeadByElpy — APPROVED** [review]

File: src/mtx3/commands/uplink/getCurrentStatusMeter.ts [hunk 5]
```diff
@@ -0,0 +1,305 @@
...
+
+import * as command from '../../../mtx/utils/command.js';
+import {READ_ONLY} from '../../../mtx/constants/accessLevels.js';
+import {IEventStatus, eventStatusMask} from '../../../mtx/utils/CommandBinaryBuffer.js';
+import CommandBinaryBuffer, {ICommandBinaryBuffer} from '../../utils/CommandBinaryBuffer.js';
+import * as bitSet from '../../../utils/bitSet.js';
+import * as types from '../../types.js';
+
+
+interface ITariff {
```
Comment (Pnom) [inline src/mtx3/commands/uplink/getCurrentStatusMeter.ts:70]: rename fields
`
interface ITariffs {
    'A+': types.TUint8;

    maximumPowers: types.TUint8;

    /**
     * Tariff for the group of generated energies (`3rd` grid).
     */
     'A-': types.TUint8;
}

`

File: src/mtx3/commands/uplink/getCurrentStatusMeter.ts [hunk 6]
```diff
@@ -0,0 +1,305 @@
...
+
+    /**
+     * Calibration enable flag.
+     */
+    calEnableFlag: boolean;
+
+    /**
+     * Current tariffs for the 1st to 3rd tariff grids.
+     */
+    curTariff: ITariff;
```
Comment (Pnom) [inline src/mtx3/commands/uplink/getCurrentStatusMeter.ts:150]: curTariff > currentTariffs

File: src/mtx/commands/uplink/getCurrentStatusMeter.ts [hunk 7]
```diff
@@ -203,20 +231,37 @@ export const examples: command.TCommandExamples = {
...
+    const tbadPmaxAll = buffer.getUint32();
+
+    // reserved
+    buffer.getUint32();
+
+    const tbadFREQ = buffer.getUint32();
+    const relayStatus = bitSet.toObject(extendedCurrentValues2RelayStatusMask, buffer.getUint8()) as unknown as IExtendedCurrentValues2RelayStatus;
+    const statusEvent1 = buffer.getUint8();
+    const statusEvent2 = buffer.getUint8();
+    const calEnableFlag = bitSet.toObject(calibrationEnableFlagMask, buffer.getUint8()) as unknown as ICalibrationEnableFlagParameters;
```
[the remark this answers is not in the document]
Comment (aleks-and) [reply src/mtx/commands/uplink/getCurrentStatusMeter.ts]: applied

File: src/mtx3/commands/uplink/getCurrentStatusMeter.ts (hunk 5 above)
Comment (aleks-and) [reply src/mtx3/commands/uplink/getCurrentStatusMeter.ts]: applied

File: src/mtx3/commands/uplink/getCurrentStatusMeter.ts (hunk 6 above)
Comment (aleks-and) [reply src/mtx3/commands/uplink/getCurrentStatusMeter.ts]: applied

**Review by Pnom — APPROVED** [review]

**Pull request merged**